app/src/main/java/org/projectbuendia/client/ui/chart/PatientChartActivity.java

```diff
@@ -271,6 +281,28 @@ public void onPageFinished(WebView view, String url) {
                 mController.onAddTestResultsPressed();
             }
         });
+
+        initChartMenu();
+    }
+
+    private void initChartMenu() {
+        List<Chart> charts = mController.getCharts();
+        String[] menuArray = new String[charts.size()];
+        int i = 0;
```

Comment:
Yes, I know that. I did that just to test and forgot to rewrite the loop. I'll fix that.